Src/VimCore/VimSettings.fs

```diff
@@ -725,16 +729,25 @@ type internal EditorToSettingSynchronizer
                 GetVimSettingValue = SettingSyncData.GetSettingValueFunc LocalSettingNames.ExpandTabName true
                 IsLocal = true
             })
-           
+
         _settingList.Add(
             {
-                EditorOptionKey = DefaultTextViewHostOptions.LineNumberMarginId.Name
-                GetEditorValue = SettingSyncData.GetBoolValueFunc DefaultTextViewHostOptions.LineNumberMarginId
+                EditorOptionKey = LineNumbersMarginOptions.NumberOptionName

```

Comment:
Why change this key here? One feature of VsVim is letting users decide if settings from vsvimrc or Visual Studio should win during startup. Using the standard editor key here is necessary for implementing this feature. With a different key they will be seen as different options.

Reply:
We can't use default value here as It'd enable the native VS line numbers margin whenever "Number" is set to true. Even if both "Number" and "RelativeNumber" are set to true (the case which is not supported by the native margin). So we need to do something different here.

I gave it a second thought and I think that we could add do 2 mappings here.
If VS wins and native margin is shown - we set "number" (so the user has at least some numbers margin) and let vim decide what to do with relativenumber. If VS wins and native margin is hidden - we set both "number" and "relativenumber" to false. 

If Vim wins - we use existing logic to hide/show margins.

Also we could show native margin whenever number is true and relativenumber is false (instead of using our custom one). This would eliminate the need of "Enable"/"Disable" settings for the new margin and also would let users have the "old" behavior (with native margin) if they don't set relativenumber to true. I'll try to see if there are any problems with this approach and implement it.